models/cow_protocol/ethereum/cow_protocol_ethereum_app_data.sql

```diff
@@ -1,5 +1,5 @@
 {{ config(alias='app_data',
-        post_hook='{{ expose_spells_hide_trino(\'["ethereum"]\',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
```

Comment:
The problem is that if we expose this as is on DuneSQL, it won't work.
 `select * from cow_protocol_ethereum.app_data limit 1;` throws `Unknown type: numeric at line 1, position 15` - could you fix the underlying issue before exposing it?

Reply:
isn't numeric supported? The issue before was that data wasn't queryable by dunesql, the raw data is queryable now

Reply:
@bh2smith Would bigint be enough for this parameter?

`BIGINT[#](https://trino.io/docs/current/language/types.html#bigint)

A 64-bit signed two’s complement integer with a minimum value of -2^63 and a maximum value of 2^63 - 1.`

Reply:
I believe this value can be just a regular integer. It's bonded between 0 and 10^5.

Reply:
@bh2smith it's[ integer now ](https://github.com/duneanalytics/spellbook/pull/2799/commits/5697a0338e18024636f78a3466e63f6b52bae601)